Schema example changes (#216)
bufbuild/docs.buf.build: docs/reference/inputs.md

The various I/O options for `buf` may seem a bit daunting, so we'll break down
how this all fits together.

> In general, a Buf schema input is a collection of `.proto` files used by many of the

Author: ```suggestion
In general, a schema input for `buf` is a collection of `.proto` files used by many of the
```

$ buf breaking --against image.bin
```

> ## Specifying a schema

Reviewer: in this section “Buf schemas” are almost referred to like a first class citizen. like a "buf schema" is almost an entity deserving of a independent definition. Would it be better to remove the word "Buf" and obtain context in another way?

> Buf schemas are almost always specified as the first argument on the command line, and with the

Reviewer: ```suggestion
Schemas are almost always specified as the first argument on the command line, and with the
```

Reviewer: maybe adding before this...

```
In this context, "schemas" refer to...
```

Reviewer: oh its in "terminology" - thats good enough.

`--against` flag for the compare against schema on `buf breaking`.

For each of `buf {build,lint,breaking,generate,ls-files}`, the schema is
specified as the first argument. Schemas are specified as a string and have this
structure:

```
path#option_key1=option_value1,option_key2=option_value2
```

> The path specifies the path to the schema. The options specify options to

Author: ```suggestion
The path specifies the path of the input flag or argument. The options specify options to
```

interpret the schema at the path.

### format option

> The `format` option can be used on any schema to override the derived

Author: ```suggestion
The `format` option can be used on any flag or argument to override the derived
```

> The `format` option can be used on any schema string to override the derived

Author: ```suggestion
The `format` option can be used on any schema to override the derived
```
